cmd/platform.go

```diff
@@ -166,12 +169,14 @@ func createGithubClient(flag *flag.FlagSet, verifyFlags bool, readOnly bool) (mu
 		BaseURL:             gitBaseURL,
 		TransportMiddleware: http.NewLoggingRoundTripper,
 		RepoListing: github.RepositoryListing{
-			Organizations:    orgs,
-			Users:            users,
-			Repositories:     repoRefs,
-			RepositorySearch: repoSearch,
-			Topics:           topics,
-			SkipForks:        skipForks,
+			Organizations:           orgs,
+			Users:                   users,
+			Repositories:            repoRefs,
+			RepositorySearch:        repoSearch,
+			Topics:                  topics,
+			SkipForks:               skipForks,
+			RepositoryIncludeFilter: repoIncludeFilter,
```

Comment:
I don't know why I didn't think of this before.

You could (if you wanted to) use `regex.Compile` here and convert the expression before assigning them to the structure. That way the error would appear more like a validation error instead of using a `panic`.